pkg/gb/instructions_table.go

```diff
@@ -0,0 +1,1444 @@
+
+		},
+		0xFE: func() {
+			// CP A,#
+			gb.instCp(gb.popPC(), gb.CPU.AF.Hi())
+
+		},
+		0x3C: func() {
+			// INC A
+			gb.instInc(gb.CPU.AF.SetHi, gb.CPU.AF.Hi())
+
+		},
+		0x04: func() {
+			// INC B
+			gb.instInc(gb.CPU.BC.SetHi, gb.CPU.BC.Hi())
+
+		},
+		0x0C: func() {
+			// INC C
+			gb.instInc(gb.CPU.BC.SetLo, gb.CPU.BC.Lo())
+
+		},
+		0x14: func() {
+			// INC D
+			gb.instInc(gb.CPU.DE.SetHi, gb.CPU.DE.Hi())
+
+		},
+		0x1C: func() {
+			// INC E
+			gb.instInc(gb.CPU.DE.SetLo, gb.CPU.DE.Lo())
+
+		},
+		0x24: func() {
+			// INC H
+			gb.instInc(gb.CPU.HL.SetHi, gb.CPU.HL.Hi())
+
+		},
+		0x2C: func() {
+			// INC L
+			gb.instInc(gb.CPU.HL.SetLo, gb.CPU.HL.Lo())
+
+		},
+		0x34: func() {
+			// INC (HL)
+			addr := gb.CPU.HL.HiLo()
+			gb.instInc(func(val byte) { gb.Memory.Write(addr, val) }, gb.Memory.Read(addr))
+
+		},
+		0x3D: func() {
+			// DEC A
+			gb.instDec(gb.CPU.AF.SetHi, gb.CPU.AF.Hi())
+
+		},
+		0x05: func() {
+			// DEC B
+			gb.instDec(gb.CPU.BC.SetHi, gb.CPU.BC.Hi())
+
+		},
+		0x0D: func() {
+			// DEC C
+			gb.instDec(gb.CPU.BC.SetLo, gb.CPU.BC.Lo())
+
+		},
+		0x15: func() {
+			// DEC D
+			gb.instDec(gb.CPU.DE.SetHi, gb.CPU.DE.Hi())
+
+		},
+		0x1D: func() {
+			// DEC E
+			gb.instDec(gb.CPU.DE.SetLo, gb.CPU.DE.Lo())
+
+		},
+		0x25: func() {
+			// DEC H
+			gb.instDec(gb.CPU.HL.SetHi, gb.CPU.HL.Hi())
+
+		},
+		0x2D: func() {
+			// DEC L
+			gb.instDec(gb.CPU.HL.SetLo, gb.CPU.HL.Lo())
+
+		},
+		0x35: func() {
+			// DEC (HL)
+			addr := gb.CPU.HL.HiLo()
+			gb.instDec(func(val byte) { gb.Memory.Write(addr, val) }, gb.Memory.Read(addr))
+
+			// ========== 16-Bit ALU ===========
+		},
+		0x09: func() {
+			// ADD HL,BC
+			gb.instAdd16(gb.CPU.HL.Set, gb.CPU.HL.HiLo(), gb.CPU.BC.HiLo())
+
+		},
+		0x19: func() {
+			// ADD HL,DE
+			gb.instAdd16(gb.CPU.HL.Set, gb.CPU.HL.HiLo(), gb.CPU.DE.HiLo())
+
+		},
+		0x29: func() {
+			// ADD HL,HL
+			gb.instAdd16(gb.CPU.HL.Set, gb.CPU.HL.HiLo(), gb.CPU.HL.HiLo())
+
+		},
+		0x39: func() {
+			// ADD HL,SP
+			gb.instAdd16(gb.CPU.HL.Set, gb.CPU.HL.HiLo(), gb.CPU.SP.HiLo())
+
+		},
+		0xE8: func() {
+			// ADD SP,n
+			gb.instAdd16Signed(gb.CPU.SP.Set, gb.CPU.SP.HiLo(), int8(gb.popPC()))
+			gb.CPU.SetZ(false)
+
+		},
+		0x03: func() {
+			// INC BC
+			gb.instInc16(gb.CPU.BC.Set, gb.CPU.BC.HiLo())
+
+		},
+		0x13: func() {
+			// INC DE
+			gb.instInc16(gb.CPU.DE.Set, gb.CPU.DE.HiLo())
+
+		},
+		0x23: func() {
+			// INC HL
+			gb.instInc16(gb.CPU.HL.Set, gb.CPU.HL.HiLo())
+
+		},
+		0x33: func() {
+			// INC SP
+			gb.instInc16(gb.CPU.SP.Set, gb.CPU.SP.HiLo())
+
+		},
+		0x0B: func() {
+			// DEC BC
+			gb.instDec16(gb.CPU.BC.Set, gb.CPU.BC.HiLo())
+
+		},
+		0x1B: func() {
+			// DEC DE
+			gb.instDec16(gb.CPU.DE.Set, gb.CPU.DE.HiLo())
+
+		},
+		0x2B: func() {
+			// DEC HL
+			gb.instDec16(gb.CPU.HL.Set, gb.CPU.HL.HiLo())
+
+		},
+		0x3B: func() {
+			// DEC SP
+			gb.instDec16(gb.CPU.SP.Set, gb.CPU.SP.HiLo())
+
+		},
+		0x27: func() { /*
```

Comment:
This comment seems to have formatted badly. Could you replace the `/* ... */` with just `//` lines? I think that makes it more consistent with the rest of the code base.